src/main/isNotEmail.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  registerDecorator,
+  ValidationOptions,
+  ValidatorConstraint,
+  ValidatorConstraintInterface,
+  ValidationArguments
+} from '@hmcts/class-validator'
+
+import * as validator from 'validator'
+
+@ValidatorConstraint()
+export class IsNotEmailConstraint implements ValidatorConstraintInterface {
+
+  validate (value: any, args: ValidationArguments): boolean {
+    if (value === undefined || value === null) {
+      return true
+    } else {
+      return (value.length === 0 || !validator.isEmail(value))
+    }
```

Comment:
This is simply one disjunction:
`return !value || value.length === 0 || !validator.isEmail(value)`

Reply:
will update

Reply:
done